<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchListItemUI</class>
 <widget class="QRadioButton" name="BatchListItemUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>75</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>201</width>
    <height>75</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>201</width>
    <height>75</height>
   </size>
  </property>
  <widget class="QLabel" name="batch_package_name">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>16</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>推荐套餐1</string>
   </property>
  </widget>
  <widget class="QLabel" name="batch_package_size">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>46</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="text">
    <string>共五款软件,共20M</string>
   </property>
  </widget>
  <widget class="QLabel" name="batch_package_type">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>0</y>
     <width>49</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="batch_package_unique">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../UmInstallerAssistant.qrc"/>
 </resources>
 <connections/>
</ui>
